lib/gem-licenses.rb

```diff
@@ -1 +1,10 @@
 require 'gem_licenses'
+
+if defined?(Rails)
```

Comment:
Should live in its own file and be loaded via the above require. See how other gems do this.